Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -81,23 +118,26 @@ extension NSManagedObjectContext {
         query: _ChannelListQuery<ExtraData.Channel>?
     ) throws -> ChannelDTO {
         let dto = ChannelDTO.loadOrCreate(cid: payload.cid, context: self)
-        dto.name = payload.name
-        dto.imageURL = payload.imageURL
-        dto.extraData = try JSONEncoder.default.encode(payload.extraData)
-        dto.typeRawValue = payload.typeRawValue
-        dto.config = try JSONEncoder().encode(payload.config)
-        dto.createdAt = payload.createdAt
-        dto.deletedAt = payload.deletedAt
-        dto.updatedAt = payload.updatedAt
-        dto.defaultSortingAt = payload.lastMessageAt ?? payload.createdAt
-        dto.lastMessageAt = payload.lastMessageAt
-        dto.memberCount = Int64(clamping: payload.memberCount)
-        
-        dto.isFrozen = payload.isFrozen
-        
-        if let createdByPayload = payload.createdBy {
-            let creatorDTO = try saveUser(payload: createdByPayload)
-            dto.createdBy = creatorDTO
+        
+        if dto.changeHash != payload.changeHash {
```

Comment:
Do we really need to come up with this logic ourselves? Doesn't `hasChanges` from core data check if all properties weren't changed? @b-onc